test/utils/image/manifest.go

```diff
@@ -216,7 +216,7 @@ const (
 
 func initImageConfigs(list RegistryList) (map[ImageID]Config, map[ImageID]Config) {
 	configs := map[ImageID]Config{}
-	configs[Agnhost] = Config{list.PromoterE2eRegistry, "agnhost", "2.53"}
+	configs[Agnhost] = Config{list.PromoterE2eRegistry, "agnhost", "2.54"}
```

Comment:
There is no `2.54` version present!
```
% manifest-tool inspect registry.k8s.io/e2e-test-images/agnhost:2.54
ERRO[0000] registry.k8s.io/e2e-test-images/agnhost:2.54: not found
ERRO[0000] Unknown descriptor type:
```

Reply:
It's being updated in #125489. That's why I'm holding this PR. I'm sorry for confusing.